release-notes/unreleased/LUN-427-optimize-for-tablet.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: LUN-427
+show_in_stores: true
+platforms:
+  - android
+  - ios
+de: Lunes wurde für Tablets optimiert
```

Comment:
Maybe `Optimierung für Tablets`. It's somehow seems strange to me to have the app name in a release note.